### PR TITLE
Added new page explaining how to a question form on registration

### PR DESCRIPTION
**Pull request by Haltarys**
https://trello.com/c/BsxB26r4

### CODE REVIEW

**Comment by JulienPradet** [comment]
Could you please update this branch with the latest version available in `master`, re-read your whole page with your new knowledge and check that everything is fine on the deployed preview?

I'll review when this is done. :slightly_smiling_face:

File: source/docs/magento2/add-question-on-registration.md [hunk 1]
```diff
@@ -3,85 +3,33 @@ id: add-question-on-registration
 title: Add a question field when creating an account
 ---
 
-Here you will learn how to add a `Question` form for the user to fill when creating an account. You may want to check [this](/docs/magento2/add-new-attribute.html) page first, if you don't know how to add an attribute and create a Front-Commerce module.
+Here you will learn how to add a `Question` form that the user can fill when creating an account. You may want to check [this](/docs/magento2/add-new-attribute.html) page first, if you don't know how to add an attribute and create a Front-Commerce module.
```
Comment (mo-fc) [inline source/docs/magento2/add-question-on-registration.md:6]:
```suggestion
Here you will learn how to add a `Question` form that the user can fill when creating an account. You may want to check [how to add a new attribute docs](/docs/magento2/add-new-attribute.html) first, if you don't know how to add an attribute and create a Front-Commerce module.
```

File: source/docs/magento2/add-question-on-registration.md [hunk 2]
```diff
@@ -3,85 +3,33 @@ id: add-question-on-registration
...
+
+![Alt: screenshot of the register form with a `Question` field](./assets/without-question-screenshot.png)
 
 (We will assume you already have done the [first](/docs/magento2/add-new-attribute.html) part.)
 
-## Add the form to the page
+## Create a server module
 
-The source code for the `Account creation` page is located at `node_modules/front-commerce/src/web/theme/modules/User/RegisterForm/`. Copy it into your module. (If you don't know how to locate the source code for a page, click [here](/docs/magento2/add-new-attribute.html#How-to-find-which-files-to-edit).)
+Create a server module, if you haven't already. In this example, ours we be called `acme`.
```
Comment (mo-fc) [inline source/docs/magento2/add-question-on-registration.md:16]:
```suggestion
Create a server module, if you haven't already. For the sake of this example, we will call it `acme`.
```

File: source/docs/magento2/add-question-on-registration.md [hunk 3]
```diff
@@ -3,85 +3,33 @@ id: add-question-on-registration
...
 
 ```bash
-mkdir -p mymodule/web/theme/modules/User/
-cp -r node_modules/front-commerce/src/web/theme/modules/User/RegisterForm/ \
-mymodule/web/theme/modules/User/
+mkdir -p mymodule/server/modules/acme
 ```
 
-Edit `RegisterForm.js` as shown below. This adds the actual form field seen on the page.
+(We will assume you already have done the first part. Click [here](/docs/magento2/add-new-attribute.html) if you haven't already.)
```
Comment (mo-fc) [inline source/docs/magento2/add-question-on-registration.md:22]:
```suggestion
Follow the steps outlined in [add a module to Front Commerce](/docs/magento2/add-new-attribute.html#Add-the-module-to-Front-Commerce) docs
```

**Comment by real34** [comment]
Closing as part of a cleanup. The content could be referred to if the question arises, and this kind of content would need more work in the future to be part of the documentation.

Sorry for not merging, after all this time and thanks again everyone for your work on this.
It'll definitely be useful when time will come!

**Pull request closed without merging**